plugins/optimization-detective/storage/data.php

```diff
@@ -200,6 +200,7 @@ function od_get_current_url_metrics_etag( OD_Tag_Visitor_Registry $tag_visitor_r
 	}
 
 	$data = array(
+		'xpath_version'    => 2, // Bump whenever a major change to the XPath format occurs so that new URL Metrics are proactively gathered.
```

Comment:
Just curious: Will this need another bump once we switch from the transition format to the new format? Or is this only relevant for the new format, which now already is being stored?

Reply:
Should we maybe set it to `1`, just because it better aligns with the current version? And because it wasn't there before its mere presence should trigger an invalidation of existing metrics anyway, correct?

Reply:
I was thinking this should be considered more like `$wp_db_version` where it is just an incremented integer. Originally I had it set to `OPTIMIZATION_DETECTIVE_VERSION` but there's no need for the ETag to change with each version unless there's something fundamentally changed about the format, like with the change here to the XPath structure.

Reply:
> Will this need another bump once we switch from the transition format to the new format? Or is this only relevant for the new format, which now already is being stored?

We won't need to bump it after the transitional period because at that point the URL Metrics will all have the new XPaths stored, so then it's just a matter of actually using them as opposed to the transitional form (🐵).

Reply:
> Should we maybe set it to `1`, just because it better aligns with the current/upcoming version? And because it wasn't there before its mere presence should trigger an invalidation of existing metrics anyway, correct?

Right, it could be anything since anything would be different than nothing. But I thought `2` made sense since it's the second version after the initial format we had been using.

Reply:
I would slightly prefer `1` (you could argue the previous one was `0`, just like the release versions have been `0.*`). But either works for me.